CHANGELOG.rst

```diff
@@ -16,3 +16,22 @@ CHANGELOG appropriately with each PR
 Release notes for previous/upcoming versions can be found in docs/source/releases,
 for reference.
 
+Enhancements
```

Comment:
Just remove these changes when you are ready to merge.  No release notes internal to template_basic_plugin (since it needs to serve as a template)... Make sure details are all included directly in the PR Summary.